server/index.js

```diff
@@ -0,0 +1,27 @@
+import express from 'express';
+import path from 'path';
+
+import router from './routes';
+
+const PORT = process.env.PORT || 3000;
+const app = express();
+
+app.use(express.json());
+app.use(express.static(path.resolve(__dirname, '..', 'dist')));
+
+// eslint-disable-next-line no-unused-vars
+app.use((err, req, res, next) => {
```

Comment:
Could you elaborate a bit on error handling if you ought to improve it? If the backend housing many features, say, for a e-commerce platform.

Reply:
I would use the node `Error` object and modify it to include HTTP status codes that match the error situation (Not found, Unauthorized, etc), and I would include a human-readable message that the frontend could display to the user. I also just remembered I should have been passing the error in `next(error)` instead of throwing them in the routes file 😅